src/devices/video/ef9345.h

```diff
@@ -59,6 +60,12 @@ class ef9345_device :   public device_t,
 	inline void inc_x(uint8_t r);
 	inline void inc_y(uint8_t r);
 
+	enum
```

Comment:
If this is a closed set of values it should be a C++11 scoped enum (`enum class`).